src/components/common/NameRequestInfo.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <div id="name-request-summary">
+    <v-row>
+      <v-col id="name-request-info" cols="4">
+        <ul>
+          <li>
+            <label>
+              <strong>Name Request</strong>
+            </label>
+          </li>
+          <li>Entity Type: {{ entityTypeDescription() }}</li>
+          <li>Request Type: {{ requestType() }}</li>
+          <li>Expiry Date: {{ formattedExpirationDate() }}</li>
+          <li>
+            <v-icon v-if="nameRequestDetails.status === NameRequestStates.APPROVED"
+              color="green" class="nr-status-icon">mdi-check</v-icon>
+            Status: {{  nameRequestDetails.status | capitalize }}
+          </li>
+          <li id="condition-consent">
+            <v-icon v-if="conditionConsent() === RECEIVED_STATE || conditionConsent() === NOT_REQUIRED"
+              color="green" class="nr-status-icon">mdi-check</v-icon>
+            <v-icon v-if="conditionConsent() === NOT_RECEIVED_STATE" color="red"
+              class="nr-status-icon">mdi-close</v-icon>
+            Condition/Consent: {{ conditionConsent() }}
+          </li>
+        </ul>
+      </v-col>
+      <v-col id="name-request-applicant-info" cols="8">
+        <ul>
+          <li>
+            <label>
+              <strong>Name Request Applicant</strong>
+            </label>
+          </li>
+          <li>Name: {{ applicantName() }}</li>
+          <li>Address: {{ applicantAddress() }}</li>
+          <li>Email: {{ nameRequestApplicant.emailAddress }}</li>
+          <li>Phone: {{ nameRequestApplicant.phoneNumber | VMask("(###) ###-####") }}</li>
+        </ul>
+      </v-col>
+    </v-row>
+  </div>
+</template>
+<script lang="ts">
```

Comment:
I like a blank line between the template and the script. (And also between the script and the styles, but you have that one :) )

Reply:
Done